azure-ts-aks-multicluster/index.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
+
+import * as azure from "@pulumi/azure";
+import * as pulumi from "@pulumi/pulumi";
+
+// Global Config
+const config = new pulumi.Config();
+const configPassword = config.require("password");
+const configSshPublicKey = config.require("sshPublicKey");
+const configResourceGroupLocation = config.require("resourceGroupLocation");
```

Comment:
It'd be good to include this required config value in the `Pulumi.yaml`. That way:
 - If the example is used with `pulumi new` (e.g. `pulumi new https://github.com/pulumi/examples/azure-ts-aks-multicluster`), the CLI will prompt the user for the value as part of setting up the stack.
 - Or if the "Deploy with Pulumi" button in the README.MD is clicked, the config value will be included in the form that the user fills out in the browser to configure the project/stack.

Otherwise, when `pulumi new` runs the initial `up` as part of its operation, it will fail due to a missing required config value.